src/zenml/post_execution/pipeline_run.py

```diff
@@ -152,13 +148,16 @@ def enable_artifact_metadata(self) -> Optional[bool]:
         Returns:
             True if artifact metadata is enabled for this pipeline run.
         """
-        from zenml.pipelines.base_pipeline import (
-            PARAM_ENABLE_ARTIFACT_METADATA,
-        )
+        return self.model.pipeline_configuration.enable_artifact_metadata
 
-        return self.model.pipeline_configuration.get(
-            PARAM_ENABLE_ARTIFACT_METADATA
-        )
+    @property
+    def enable_artifact_visualization(self) -> Optional[bool]:
+        """Returns whether artifact visualization is enabled for this run.
+
+        Returns:
+            True if artifact visualization is enabled for this pipeline run.
+        """
+        return self.model.pipeline_configuration.enable_artifact_visualization
```

Comment:
Should this just return a bool rather than an optional bool? Think we do it that way in other places, rather than giving people a possible `None` return value from a method whose name implies a True vs False output.

Reply:
Currently, it is like this to be consistent with `enable_cache`, ... which all return Optional as well. Perhaps we can change this once we do the post execution redesign in one of the next sprints